packages/template-retail-react-app/app/pages/product-detail/index.test.js

```diff
@@ -84,31 +91,41 @@ test('should render product details page', async () => {
     global.server.use(
         // Use a single product (and not a product set)
         rest.get('*/products/:productId', (req, res, ctx) => {
-            return res(ctx.json(productsResponse.data[0]))
+            return res(ctx.json(masterProduct))
```

Comment:
We pushed history to be a master product, we should mock the data accordingly. the `productsResponse.data[0]` we used previously was a variant product, not a master one. Therefore, although the tests passed, but it caused a lot of react warning errors during the tests. This would fix these errors